core/src/SYCL/Kokkos_SYCL_Team.hpp

```diff
@@ -111,8 +111,8 @@ class SYCLTeamMember {
   template <class ValueType>
   KOKKOS_INLINE_FUNCTION void team_broadcast(ValueType& val,
                                              const int thread_id) const {
-    val = sycl::ONEAPI::broadcast(m_item.get_group(), val,
-                                  sycl::id<2>(thread_id, 0));
+    val = sycl::group_broadcast(m_item.get_group(), val,
```

Comment:
`sycl::ONEAPI::broadcast` is deprecated in the newer release.

Reply:
Does the other one exist in previous releases?

Reply:
When `sycl::group_broadcast` was introduced `sycl::ONEAPI::broadcast` was deprecated, see https://github.com/intel/llvm/pull/3786.

Reply:
OK so this means that when we merge we require a more recent version of the Intel toolkit.

Reply:
Yes, my strategy is to basically go with the latest release on the early access machines. (Otherwise, we can also disable deprecation warnings but I don't prefer that.)